lib/transports/elasticsearch.js

```diff
@@ -93,6 +93,11 @@ elasticsearch.prototype.getData = function (limit, offset, callback){
 
                 var body = jsonParser.parse(response.body);
                 self.lastScrollId = body._scroll_id;
+                if(self.lastScrollId === undefined){
```

Comment:
Maybe you can forward the error information included in the body, and if there is none, I will be happy to file a bug with elasticsearch requesting it.